src/psyclone/psyir/transformations/acc_kernels_trans.py

```diff
@@ -77,12 +77,12 @@ class ACCKernelsTrans(RegionTrans):
     excluded_node_types = (CodeBlock, Return, PSyDataNode,
                            psyGen.HaloExchange, WhileLoop)
 
-    def apply(self, node, options=None):
+    def apply(self, node: Union[Node, List[Node]], options: dict = None):
```

Comment:
I'm not sure if typing this is intentional or not - I've not seen this anywhere in PSyclone previously - if it is a new plan and I've missed it then let me know.